tools/maybe-shim-windows.js

```diff
@@ -0,0 +1,28 @@
+const fs = require('fs');
+
+/*
+ * Adds a shim to fix Windows symlinking with create-electron-app.
+ * Should run on Windows only.
+ * More details: https://github.com/boltpkg/bolt/issues/207
+ */
+function createShim(dir, shimPath) {
+  const cwd = process.cwd();
+  fs.mkdirSync(`${cwd}/${dir}`, { recursive: true });
+
+  const data = new Uint8Array(Buffer.from('shim file'));
+  fs.writeFileSync(`${cwd}/${shimPath}`, data);
```

Comment:
```suggestion
  fs.writeFileSync(`${cwd}/${shimPath}`, '');
```

We can write an empty string